tests/example_9_nuclear_dynamics/test_ethd_2D.py

```diff
@@ -0,0 +1,428 @@
+#*********************************************************************************
+#* Copyright (C) 2017-2018  Brendan A. Smith, Alexey V. Akimov
+#*
+#* This file is distributed under the terms of the GNU General Public License
+#* as published by the Free Software Foundation, either version 2 of
+#* the License, or (at your option) any later version.
+#* See the file LICENSE in the root directory of this distribution
+#* or <http://www.gnu.org/licenses/>.
+#*
+#*********************************************************************************/
+
+
+import cmath
+import math
+import os
+import sys
+
+if sys.platform=="cygwin":
+    from cyglibra_core import *
+elif sys.platform=="linux" or sys.platform=="linux2":
+    from liblibra_core import *
+
+from libra_py import *
+import aux_functs
+
+class tmp:
+    pass    
+
+def model1(q, params, full_id):

```

Comment:
Please, include this model as one of the models (e.g. 5) into the models_Libra module